test/integration/docker-compose-gosql.yml

```diff
@@ -0,0 +1,89 @@
+version: '3.8'
+
+services:
```

Comment:
I'm really confused about which test files are active here. It looks like this template has Beyla, and also it's own instance of the otel-collector, jaeger etc. But then we also have similar components as part of the OATS framework. Seems to me this should really be a very simple docker compose file, only containining the sql client?

Reply:
Looks like this was accidentally included in the commits, will remove.  Good catch!